src/pages/wallet/add.custom.token.confirm.js

```diff
@@ -158,7 +158,7 @@ class AddCustomToken extends Component {
         }
         const result = await CancelablePromiseUtil.makeCancelable(parseHelper.getUserTokenBalance(type, chain, contractAddress, address), this);
         console.log('UserTokenBalance: ', result);
-        this.setState({ balance: common.weiToCoin(result.balance) });
+        this.setState({ balance: common.weiToCoin(result.balance, this.decimals) });
```

Comment:
naming inconsistency - using `decimals` here but `precision` elsewhere. let's change all instances to use the same variable name.